src/components/tests/CharacterDetails.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { screen, render } from '@testing-library/react'
```

Comment:
I'd personally prefer to set tile file under the same level of original implementation such as below;

```
// no need for tests folder
CharacterDetails.tsx
CharacterDetails.test.tsx
CharacterItem.tsx
CharacterItem.test.tsx
[...]
```

And the mock data can be close to where its original value is defined. In this case, you can declare `modifiedCaracter`(mocks/modifiedCharacters.tsx) under API folders as that's where you define it.